commons/zenoh-core/src/macros.rs

```diff
@@ -115,14 +115,14 @@ macro_rules! zasyncrecv {
 
 // This macro checks the boolean results of an operation and returns in case
 // the result is false. Basically, it implements the ? operator for booleans
-#[macro_export]
-macro_rules! zcheck {
-    ($op:expr) => {
-        if !$op {
-            return false;
-        }
-    };
-}
+// #[macro_export]
+// macro_rules! zcheck {
+//     ($op:expr) => {
+//         if !$op {
+//             return false;
```

Comment:
commented code might as well get deleted

Reply:
I forgot about that piece of code, but I re-found it literally 5 minutes ago and removed in the last commit already 😃